fcp/arguments.py

```diff
@@ -0,0 +1,25 @@
+'''
+Common argparse arguments for each script.
+'''
+
+import argparse
+
+import fcp.node
+
+def timeout_type(time_str):
+    '''
+    Argparse argument type for timeouts.
+    '''
+    try:
+        return fcp.node.parseTime(time_str)
+    except:
+        raise argparse.ArgumentTypeError("Invalid timeout: %s" % time_str)
+
+def add_default_arguments(parser):
+    '''
+    Add the default arguments to an argparse parser.
+    '''
+    parser.add_argument('--version', '-V', action='version', version="%(prog)s " + fcp.node.fcpVersion)
+    parser.add_argument('--fcphost', '-H', default=fcp.node.defaultFCPHost, help='Connect to FCP service at host <FCPHOST>. (You may also use the environment variable FCP_HOST)')
+    parser.add_argument('--fcpport', '-P', default=fcp.node.defaultFCPPort, type=int, help='Connect to FCP service at port <FCPPORT>. (You may also use the environment variable FCP_PORT)')
+    parser.add_argument('--timeout', '-t', default=fcp.node.ONE_YEAR, type=timeout_type, help='Set the timeout, in seconds, for completion. Default one year')
```

Comment:
These lines seem rather long - they're a tight fit at least on my monitor - and [PEP8 suggests](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) a line length limit of 79 characters. What do you think of 5fef46d91941ce7ed8569e707786528e00ba6c9d? It also adds full stops, which is arguably poor practice on my part, but anyway argparse removes extra whitespace so it does not add gaps to the `--help` output.